domain/storage/service/service.go

```diff
@@ -38,7 +45,7 @@ func NewService(st State, logger logger.Logger, registryGetter storage.ModelStor
 // Deprecated: This method will be removed once the storage registry is fully
 // implemented in each service.
 func (s *Service) GetStorageRegistry(ctx context.Context) (internalstorage.ProviderRegistry, error) {
-	registry, err := s.registryGetter.GetStorageRegistry(ctx)
+	registry, err := s.StorageService.registryGetter.GetStorageRegistry(ctx)
```

Comment:
This feels like a smell to me.

Reply:
The whole method is going away as soon as the facade cut over is done. 
It's marked as deprecated for that reason. But for now we need it to serve the facade.